### ISSUE TITLE
Proof-reading a lesson and adding a new issue describing something to be improved

### ISSUE THREAD

**Issue by sarah0809** [datacarpentry/spreadsheet-ecology-lesson#287]
Data Organization in Spreadsheets for Ecologists

- Introduction

	Spreadsheet outline:
	Understand approaches for handling dates in spreadsheets --> handle dates in spreadsheets efficiently

	Spreadsheet programs:
	Maybe I overlooked this, but it would be good to mention which version of Excel was used for the exercises/screenshots.

	Problems with spreadsheets:
	To me it is not very clear what is meant with 'We advise you to do this sort of operation within your document editing software.'

- Dates as data

	Objectives
	After this episode you will be able to
		--> effectively store and format dates in spreadsheets
		--> apply alternative date formatting in spreadsheets
		--> demonstrate best practices for entering dates in spreadsheets

	Dates stored as integers
	TYPO: We can then add values to DO that

- Exporting Data

	Caveats on commas
	I think here it would be worth mentioning an example e.g. that commas are used as 1000 separators and that this is not the same in every language - so it is important to check which language is selected in your Excel .

**Comment by hoytpr** [comment]
Hi @sarah0809 and thanks very much for your critical reading of the lessons! You've made some great suggestions, and caught a typo... Do you feel comfortable doing a pull request to fix the typo? I'm happy to provide some guidance if you need it. 

Your use of more direct wording for Objectives in "Dates as Data" are also good. This lesson does have a lot of discussion, but your wording seems more appropriate as an objective. 

Also you are correct about comma separators. This should be mentioned. If you can do the PR on the typo, we can work on the other items together.

Pete

**Comment by hoytpr** [comment]
Hi @sarah0809 . I want to re-open this so we can work through your other suggestions! You seem to be comfortable with GitHub which is great. As maintainers we try to get some consensus on changes from the community (you) and other maintainers when changes are made to some wording. So I'm alerting other maintainers for this lesson:  @cbahlai @tracykteal for feedback. If we don't hear back from them, we can go ahead and make the changes you suggest. Thanks again for your careful review of the lesson. 
Peter

**Comment by tracykteal** [comment]
Thanks @sarah0809! I won't be able to get to it today, but I'm happy to help give a review and figure out how we can incorporate these suggestions.

**Comment by hoytpr** [comment]
Okay, let's start:
1. Regarding the version of Excel used in the lessons and screenshots: The versions vary, and while that can be a little confusing, it is impossible to show all examples for all spreadsheet software. Knowing this, we have relied on the setup statement: "Commands may differ a bit between programs, but the general ideas for thinking about spreadsheets are the same.". This is true, and probably the best we can do. 

2. You indicated that you would make the following change: "Understand approaches for handling dates in spreadsheets --> handle dates in spreadsheets efficiently".  I understand and generally agree that more direct statements are preferred, but in this case, I would leave the Outline the same. My justification is that we are likely teaching beginners, and there are ***advanced*** ways to use dates "efficiently", but we are teaching basic competency. 

3. I agree the statement 'We advise you to do this sort of operation within your document editing software.' is unclear. The intended message for the learner is to use whatever software you normally use for creating a document/manuscript/report rather than a text editor. It could even include using a spreadsheet program to make the tables "pretty". The important part of the message is to keep your DATA in a format that is useful for downstream data analytics. I suggest changing this to: "To do these manipulations, use a different copy of your data, or your favorite document software." 

4. You suggested changing objectives to:
```
After this episode you will be able to
--> effectively store and format dates in spreadsheets
--> apply alternative date formatting in spreadsheets
--> demonstrate best practices for entering dates in spreadsheets
```
These are really good suggestions! In my opinion, I'd say "use best practices for entering dates in spreadsheets". Your opinions are welcome!

5. Comma separators: This is a great observation. My bias would be to recommend not using commas at all, because they would interfere with comma-delimited exports. So maybe recommending not using **any** numerical separators in your data would be best. Please let me know what you think!

Thanks,
Peter

**Comment by hoytpr** [comment]
Many of these will be addressed in issue #296 and comments are welcome.
